packages/dds/tree/src/shared-tree/treeApiAlpha.ts

```diff
@@ -0,0 +1,356 @@
+
+	importConcise<TSchema extends ImplicitFieldSchema | UnsafeUnknownSchema>(
+		schema: UnsafeUnknownSchema extends TSchema
+			? ImplicitFieldSchema
+			: TSchema & ImplicitFieldSchema,
+		data: ConciseTree | undefined,
+	): Unhydrated<
+		TSchema extends ImplicitFieldSchema
+			? TreeFieldFromImplicitField<TSchema>
+			: TreeNode | TreeLeafValue | undefined
+	> {
+		return createFromInsertable<UnsafeUnknownSchema>(
+			schema,
+			data as InsertableField<UnsafeUnknownSchema>,
+		) as Unhydrated<
+			TSchema extends ImplicitFieldSchema
+				? TreeFieldFromImplicitField<TSchema>
+				: TreeNode | TreeLeafValue | undefined
+		>;
+	},
+
+	importVerbose<const TSchema extends ImplicitFieldSchema, THandle>(
+		schema: TSchema,
+		data: VerboseTree<THandle> | undefined,
+		options?: Partial<ParseOptions<THandle>>,
+	): Unhydrated<TreeFieldFromImplicitField<TSchema>> {
+		const config: ParseOptions<THandle> = {
+			valueConverter: (input: VerboseTree<THandle>) => {
+				return input as TreeLeafValue | VerboseTreeNode<THandle>;
+			},
+			...options,
+		};
+		const schemalessConfig = applySchemaToParserOptions(schema, config);
+		if (data === undefined) {
+			const field = normalizeFieldSchema(schema);
+			if (field.kind !== FieldKind.Optional) {
+				throw new UsageError("undefined provided for non-optional field.");
+			}
+			return undefined as Unhydrated<TreeFieldFromImplicitField<TSchema>>;
+		}
+		const cursor = cursorFromVerbose<THandle>(data, schemalessConfig);
+		return createFromCursor(schema, cursor);
+	},
+
+	exportConcise<T>(
+		node: TreeNode | TreeLeafValue,
+		options?: Partial<EncodeOptions<T>>,
+	): ConciseTree<T> {
+		const config: EncodeOptions<T> = {
+			valueConverter(handle: IFluidHandle): T {
+				return handle as T;
+			},
+			...options,
+		};
+
+		const cursor = borrowCursorFromTreeNodeOrValue(node);
+		return conciseFromCursor(cursor, tryGetSchema(node) ?? fail("invalid input"), config);
+	},
+
+	exportVerbose<T>(
+		node: TreeNode | TreeLeafValue,
+		options?: Partial<EncodeOptions<T>>,
+	): VerboseTree<T> {
+		const config: EncodeOptions<T> = {
+			valueConverter(handle: IFluidHandle): T {
+				return handle as T;
+			},
+			...options,
+		};
+
+		const cursor = borrowCursorFromTreeNodeOrValue(node);
+		return verboseFromCursor(cursor, tryGetSchema(node) ?? fail("invalid input"), config);
+	},
+
+	exportCompressed(
+		node: TreeNode | TreeLeafValue,
+		options: {
+			oldestCompatibleClient: FluidClientVersion;
+			idCompressor?: IIdCompressor;
+		},
+	): JsonCompatible<IFluidHandle> {
+		const schema = tryGetSchema(node) ?? fail("invalid input");
+		const format = versionToFormat[options.oldestCompatibleClient];
+		const codec = makeFieldBatchCodec({ jsonValidator: noopValidator }, format);
+		const cursor = borrowFieldCursorFromTreeNodeOrValue(node);
+		const batch: FieldBatch = [cursor];
+		// If none provided, create a compressor which will not compress anything (TODO: is this the right way to do that?).
```

Comment:
this seems like a todo that should prob be addressed before merging

Reply:
@taylorsw04 Can you provide some inside here and on the TODO below?

Reply:
For this TODO, yes, this is correct if you want the tryRecompress method to return undefined and thus compress nothing. Regarding the one below, I don't know what originatorId is used for in the rebaser stuff (revision tags, I assume @alex-pardes) but I do know that is the correct parameter to pass. I can't speak for what it will be used for, so in the case where we just make a new compressor, I don't know what could go wrong (my guess is nothing).

Reply:
```suggestion
		// If none provided, create a compressor which will not compress anything.
```